src/coreComponents/physicsSolvers/multiphysics/PhaseFieldFractureSolver.cpp

```diff
@@ -107,33 +86,6 @@ void PhaseFieldFractureSolver::implicitStepSetup( real64 const & GEOSX_UNUSED_PA
   } );
 }
```

Comment:
@andrembcosta just to confirm, do we still need these `TotalMeanStress` stuff in the phase-field solver? I think they are useless and we can remove them.

Reply:
It does not seem to be used anywhere so I have just removed it.

Reply:
I have never used them.